pybamm/models/full_battery_models/lithium_ion/basic_dfn_half_cell.py

```diff
@@ -0,0 +1,420 @@
+#
+# Basic Doyle-Fuller-Newman (DFN) Half Cell Model
+#
+import pybamm
+from .base_lithium_ion_model import BaseModel
+
+
+class BasicDFNHalfCell(BaseModel):
+    """Doyle-Fuller-Newman (DFN) model of a lithium-ion battery with lithium counter
+    electrode, from [2]_.
+
+    This class differs from the :class:`pybamm.lithium_ion.DFN` model class in that it
+    shows the whole model in a single class. This comes at the cost of flexibility in
+    comparing different physical effects, and in general the main DFN class should be
+    used instead.
+
+    Parameters
+    ----------
+    name : str, optional
+        The name of the model.
+
+    References
+    ----------
+    .. [2] SG Marquis, V Sulzer, R Timms, CP Please and SJ Chapman. “An asymptotic
+           derivation of a single particle model with electrolyte”. Journal of The
+           Electrochemical Society, 166(15):A3693–A3706, 2019
+
+    **Extends:** :class:`pybamm.lithium_ion.BaseModel`
+    """
+
+    def __init__(
+        self,
+        name="Doyle-Fuller-Newman half cell model",
+        options={"working electrode": "negative"},
```

Comment:
I think there shouldn't be a default for this option, it should always be specified (unless negative is much more common than positive?)

Reply:
There is no default now